tools/Azure.Mcp.Tools.AzureManagedLustre/src/Commands/FileSystem/FileSystemUpdateCommand.cs

```diff
@@ -0,0 +1,140 @@
+        options.ResourceGroup ??= parseResult.GetValueOrDefault<string>(OptionDefinitions.Common.ResourceGroup.Name);
+        options.Name = parseResult.GetValueOrDefault<string>(AzureManagedLustreOptionDefinitions.NameOption.Name);
+        options.MaintenanceDay = parseResult.GetValueOrDefault<string>(AzureManagedLustreOptionDefinitions.OptionalMaintenanceDayOption.Name);
+        options.MaintenanceTime = parseResult.GetValueOrDefault<string>(AzureManagedLustreOptionDefinitions.OptionalMaintenanceTimeOption.Name);
+        options.RootSquashMode = parseResult.GetValueOrDefault<string>(AzureManagedLustreOptionDefinitions.RootSquashModeOption.Name);
+        options.NoSquashNidLists = parseResult.GetValueOrDefault<string>(AzureManagedLustreOptionDefinitions.NoSquashNidListsOption.Name);
+        options.SquashUid = parseResult.GetValueOrDefault<long?>(AzureManagedLustreOptionDefinitions.SquashUidOption.Name);
+        options.SquashGid = parseResult.GetValueOrDefault<long?>(AzureManagedLustreOptionDefinitions.SquashGidOption.Name);
+        return options;
+    }
+    public override async Task<CommandResponse> ExecuteAsync(CommandContext context, ParseResult parseResult)
+    {
+        try
+        {
+            if (!Validate(parseResult.CommandResult, context.Response).IsValid)
+            {
+                return context.Response;
+            }
+            var options = BindOptions(parseResult);
+
+            if (string.IsNullOrWhiteSpace(options.MaintenanceDay) &&
+                string.IsNullOrWhiteSpace(options.MaintenanceTime) &&
+                string.IsNullOrWhiteSpace(options.RootSquashMode))
+            {
+                context.Response.Status = 400;
+                context.Response.Message = "At least one of maintenance-day/time or root-squash fields must be provided.";
+                return context.Response;
+            }
+
+            var svc = context.GetService<IAzureManagedLustreService>();
+            var fs = await svc.UpdateFileSystemAsync(
+                options.Subscription!,
+                options.ResourceGroup!,
+                options.Name!,
+                options.MaintenanceDay,
+                options.MaintenanceTime,
+                options.RootSquashMode,
+                options.NoSquashNidLists,
+                options.SquashUid,
+                options.SquashGid,
+                options.Tenant,
+                options.RetryPolicy);
+
+            context.Response.Results = ResponseResult.Create(new FileSystemUpdateResult(fs), AzureManagedLustreJsonContext.Default.FileSystemUpdateResult);
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "Error updating AMLFS.");
+            HandleException(context, ex);
+        }
+
+        return context.Response;
+    }
+
+    protected override int GetStatusCode(Exception ex) => ex switch
+    {
+        Azure.RequestFailedException reqEx => reqEx.Status,
+        _ => base.GetStatusCode(ex)
+    };
```

Comment:
Same situation, doesn't add anything over the base implementation and would remove this.